music_album.rb

```diff
@@ -0,0 +1,14 @@
+require './item'
+
+class MusicAlbum < Item
+  attr_reader :on_spotify
+
+  def initialize(on_spotify, publish_date)
+    super(id, publish_date, archived)
```

Comment:
A good idea would be if you change the argument of the super method for removing the "id" and "archived" parameters, since the parent class (Item) doesn't take those arguments in its initializer.
```suggestion
    super(publish_date)
```